.github/actions/install-dependencies/action.yml

```
@@ -9,8 +9,12 @@ inputs:
     description: "Used only on linux distro type: debian, fedora, suse, arch"
     required: false
 
-  mac-qt-version:
-    description: "The verison of Qt to install on mac os"
+  qt-version:
+    description: "The verison of Qt to install (Windows / Mac os)"
+    required: false
+
+  qt-install-dir:
+    description: "The path to install Qt into (Windows / Mac os)"
```

Typos ("verison", "Mac os")

```suggestion
    description: "The version of Qt to install (Windows & macOS)"
    required: false

  qt-install-dir:
    description: "The path to install Qt into (Windows & macOS)"
```